packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -130,6 +130,11 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
     <>
       <div ref={messageRef}>
         <Chat.Message
+          aria-label={`${message.content}${message.senderDisplayName}${
```

Comment:
Should we have a localized string here that reasonably separates the different parts?

Something like "Message sent by ${message.senderDisplayName} at ${....timestamp}. ${message.senderDisplayName} says ${message.content}"

Reply:
When tab focus on the message, the content and display name and time stamp is announced

Reply:
cool, that sounds reasonable. Let's see if we can improve what's announced using the templated localized string as I suggested above.

Reply:
Also, what about messages sent by the local participant. Should we say "message sent by you" instead of announcing the local participant's name?

Reply:
Updated PR, what I put down is message sent by myself

Reply:
when the message is sent by myself, it gets an empty display name